python/pyarrow/tests/parquet/test_parquet_file.py

```diff
@@ -348,6 +348,7 @@ def test_read_statistics():
     statistics = pq.ParquetFile(buf).read().columns[0].chunks[0].statistics
     assert statistics.null_count == 1
     assert statistics.distinct_count is None
+    assert statistics.is_distinct_count_exact is False
```

Comment:
I'm not sure the current Python API makes a test for this easy or not, but it would be nice if codepaths for `is_distinct_count_exact` == `None` and `True` were tested too.

Reply:
How about something like:
```suggestion
    # TODO: add tests for is_distinct_count_exact == None and True once Python API allows
    assert statistics.is_distinct_count_exact is False
```

Reply:
Thanks! I've applied your suggestion.